src/main/java/org/zalando/typemapper/postgres/PgTypeHelper.java

```diff
@@ -501,17 +500,7 @@ public static String toPgString(Object o, final Connection connection) {
             } else {
                 tmpd = new Timestamp(((Date) o).getTime());
             }
-
-            if (connection instanceof BaseConnection) {
-
-                // if we do have a valid postgresql connection use this one:
-                final BaseConnection postgresBaseConnection = (BaseConnection) connection;
-                sb.append(postgresBaseConnection.getTimestampUtils().toString(null, tmpd));
-            } else {
-
-                // no valid postgresql connection - use that one:
-                sb.append(postgresJDBCDriverReusedTimestampUtils.toString(null, tmpd));
-            }
+            sb.append(postgresJDBCDriverReusedTimestampUtils.toString(null, tmpd));
```

Comment:
It seems you removed the alternative part.  This part would use the timezone which is set per connection.  How can we be sure that we can ignore the connection specific timezone and use the default timezone all the time?